### PR TITLE
Enhance and add information to building-chainblocks documentation

### PR DESCRIPTION
**Pull request by satendrapalsingh**

### CODE REVIEW

**Review by sinkingsugar — APPROVED** [review]

**Comment by satendrapalsingh** [comment]
@Kryptos-FR , @sinkingsugar  - I don't have any information on the build target `cbedn`. Shall I remove it for now? 
[this commit](https://github.com/fragcolor-xyz/chainblocks/pull/249/commits/2fd95eafd4a1f9b7b7aa5437dc17494b16a6e169)

File: docs/docs/contribute/code/building-chainblocks.md [hunk 1]
```diff
@@ -5,33 +5,39 @@ license: CC-BY-SA-4.0
 
 # Building Chainblocks
 
-*Before you start, ensure you've [set up your development environment](../getting-started/#development-environment).*
+This guide explains how to build [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) from the sources, for Windows. 
 
-This guide will outline the process to build [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) from the sources.
+??? note "For Mac/Linux Users"
+    The software requirements and terminal commands for Mac/Linux machines are the same as given for Windows. Most of these packages are pre-installed on these machines, but if required can be easily installed/updated on via `brew [install/upgrade] <package-name>` on Mac, `sudo apt-get [install/update] <package-name>` on Linux.
```
[the remark this answers is not in the document]
Comment (satendrapalsingh) [reply docs/docs/contribute/code/building-chainblocks.md:11]: Agreed - added command tabs for Mac/Linux where applicable.

File: docs/docs/contribute/code/building-chainblocks.md [hunk 2]
```diff
@@ -83,67 +113,149 @@ Run the following command from the MYSYS2 MSYS terminal to update your packages:
     .
     ```
 
-Restart the terminal (if needed) and install the required dependencies with this command:
+Restart the MingW terminal (if needed) and install the required build dependencies with this command (replace the `w64-x86_64` as appropriate for your target OS/machine).
```
[the remark this answers is not in the document]
Comment (satendrapalsingh) [reply docs/docs/contribute/code/building-chainblocks.md:118]: Fixed

File: docs/docs/blocks/index.md [hunk 3]
```diff
@@ -15,6 +15,9 @@ One can think of it as a comprehensible JSON on steroids that can be evaluated o
 The execution of Chainblocks scripts is meant to be fast: the scripting language is just a description of the computational graph.
 The runtime itself is completely detached.
 
+??? note "Running code samples"
+    Many block code samples (like [Take](/General/Take)) do not include the chain/node context they need to run. To run these samples you need to navigate to `../chainblocks/docs/samples/`and use the command `../../build/cbl.exe run_sample.edn --file "<relative-script-path.edn>" > >(tee "<relative-script-path.edn>.log")`. This command provides the required boilerplate code to these samples. Use a Git/ Linux/ WSL shell for this as Windows terminals do not support the `tee` command.
```
[the remark this answers is not in the document]
Comment (satendrapalsingh) [reply docs/docs/blocks/index.md:19]: Agreed; fixed.

**Review by Kryptos-FR — APPROVED** [review]

**Pull request merged**